src/bundler/bundle_v2.zig

```diff
@@ -7546,6 +7548,91 @@ pub const LinkerContext = struct {
         return hasher.digest();
     }
 
+    pub fn validateTLA(c: *LinkerContext, source_index: Index.Int) js_ast.TlaCheck {
```

Comment:
I suggest passing the slices in to the function, so that we avoid doing any extra work

Reply:
https://github.com/oven-sh/bun/pull/14218/commits/b6add9c5314fddbdecff3f925a6bd09fd6517e1c